waddrmgr/db.go

```diff
@@ -902,10 +904,17 @@ func deserializeImportedAddress(row *dbAddressRow) (*dbImportedAddressRow, error
 		dbAddressRow: *row,
 	}
 
-	pubLen := binary.LittleEndian.Uint32(row.rawData[0:4])
+	pkHashLen := binary.LittleEndian.Uint32(row.rawData[0:4])
+	retRow.encryptedPubKeyHash = make([]byte, pkHashLen)
+	copy(retRow.encryptedPubKeyHash, row.rawData[4:4+pkHashLen])
```

Comment:
For the record, this is not correctly bounds-checked, and may panic on a bad value.  These programming errors are all throughout this package though; we'll fix them up later.